kubejs/server_scripts/mods/AE2.js

```diff
@@ -557,50 +557,83 @@ ServerEvents.recipes(event => {
         .duration(50)
         .EUt(16)
 
-    // Fix Certus
+    //Certus
+
     event.replaceInput(
-        { type: 'ae2:item_tranformation' },
-        'gtceu:certus_quartz_gem',
+        { id: /transform/ },
```

Comment:
```suggestion
        { id: /^ae2:transform/ },
```
specify the modid on this regex, dont want to accidentally have side effects